src/ontology/uberon-edit.obo

```diff
@@ -224008,6 +224008,53 @@ relationship: part_of UBERON:0000101 ! lobe of lung
 relationship: proximally_connected_to UBERON:0002184 ! segmental bronchus
 property_value: dcterms-date "2023-03-14T14:54:15Z" xsd:dateTime
 
+[Term]
+id: UBERON:8600010
+name: bronchial submucosal gland ciliated duct
+def: "A ciliated duct in a bronchial submucosal gland." [PMID:30864819]
+intersection_of: UBERON:8600014 ! submucosal gland ciliated duct
+intersection_of: part_of UBERON:0002185 ! bronchus
```

Comment:
Consider breaking this up into subclasses:

'submucosal gland ciliated duct'
'part of' some ‘submucosa of bronchus’

Reply:
Alternatively, consider relating this directly to the 'bronchus submucosal gland' with the following definition:

duct and ‘part of’ some ‘bronchus submucosal gland’  and (‘has characteristic’ some ciliatedness)
 or
duct and ‘part of’ some ‘bronchus submucosal gland’  and ('has part' some 'ciliated epithelial cell')

Reply:
I don't agree with these changes.

> 'part of' some ‘submucosa of bronchus’

It is not part of the submucosa of bronchus. Submucosal glands are located in submucosa, but they are not part of them. 
i agree that in many terms the logical axions are too restrictive, but in this case I think it is appropriate, as any submucosal gland ciliated duct that is located in the bronchus, will be a bronchial submucosal gland ciliated duct.

> duct and ‘part of’ some ‘bronchus submucosal gland’ and ('has part' some 'ciliated epithelial cell')

I can add this to the logical axiom (duct I would to the parental terms), but nor replacing with `submucosal gland ciliated duct`. Collecting ducts have also been describe to have collecting ducts (although authors could be describing the ciliated duct but not discriminating).

Reply:
> It is not part of the submucosa of bronchus. Submucosal glands are located in submucosa, but they are not part of them.

Of note, Uberon seems to have inconsistencies related to this (see below).

UBERON:0005204 ‘larynx submucosa gland’
gland and (**'part of'** some 'larynx submucosa')

vs

UBERON:0011148 ‘submucosal gland’
'mucous gland' and (**'located in'** some submucosa)

Regarding definition: if keeping the current logical definition, the text definition, if aligned, would be:

"A submucosal gland ciliated duct that is part of the bronchus."

However, this modelling skips a level of anatomical hierarchy (from the submucosal gland to the bronchus), when the immediate structure (‘bronchus submucosal gland') is available. In this case, querying parts of the 'bronchus submucosal gland' would not include this structure, which seems to be an oversight.

Reply:
> However, this modelling skips a level of anatomical hierarchy (the submucosal gland), when the immediate structure (‘bronchus submucosal gland') is available. In this case, querying parts of the 'bronchus submucosal gland' would not include this new term, which seems to be an oversight.

That is strange, I would have expected it to be inferred. Ok, I will change bronchus for bronchus submucosal gland, thank you.

>UBERON:0005204 ‘larynx submucosa gland’
gland and ('part of' some 'larynx submucosa')

That is wrong and it should be changed.

Reply:
> has_part CL:0000067 ! ciliated epithelial cell

This is probably redundant since it is already inherited from 'submucosal gland ciliated duct'

Reply:
I'm not sure it is.
If X has part Y
and Z is part of X
I don't think Z has part necessarily of Y.

Example: my body has part my brain. My arm is part of my body. But my arm does not have part my brain.